wp-job-manager-functions.php

```diff
@@ -723,6 +723,10 @@ function job_manager_upload_file( $file, $args = array() ) {
 		$allowed_mime_types = $args['allowed_mime_types'];
 	}
 
+	if( is_wp_error( $file = apply_filters( 'job_manager_upload_file_pre_upload', $file, $args, $allowed_mime_types ) ) ) {
```

Comment:
Incredibly picky, but can you add the space after the `if`?

I'm not crazy about the readability of the line, but not a blocker for me.